### PR TITLE
LPS-166394/LPS-166393/LPS-166392/LPS-166391/LPS-168347

### PR DESCRIPTION
**Pull request by lucasperj**
Issues: 
https://issues.liferay.com/browse/LPS-166394
https://issues.liferay.com/browse/LPS-168347
https://issues.liferay.com/browse/LPS-166393
https://issues.liferay.com/browse/LPS-166392
https://issues.liferay.com/browse/LPS-166391

### CODE REVIEW

**Comment by lucasperj** [comment]
ci:test:upgrade

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/UpgradeReport.testcase [hunk 1]
```diff
@@ -215,12 +258,36 @@ definition {
 	}
 
 	@description = "LPS-154683: The Upgrade Report logs total upgrade runtime."
-	@ignore = "Test Stub"
+	@ignore = "true"
```
Comment (lucasperj) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/UpgradeReport.testcase:261]: Blocked by : [LPS-166394](https://issues.liferay.com/browse/LPS-168372)

**Comment by lucasperj** [comment]
Apparently DocumentLibraryLogged have not changed the size of the DL running on CI.

**Comment by vicnate5** [comment]
@lucasperj Glad to hear CI is reporting results consistent with local runs. Nice job making use of the RegexUtil.

Can you updates with some changes?
1. Update tests to only print/echo the `fileContent` if the test fails. E.g. remove https://github.com/vicnate5/liferay-portal/pull/2192/files#diff-9670414131aeeaaff4001b58455f07d7ab5aaa93fd0967e9f46d38bbd3df54f0R89 and move it into the `fail` conditional.
2. Add context for `echo("${upgradeRunTime}");`. When printing out a value, make sure it is explained in the printout. e.g. `Upgrade runtime reported by upgrade.log is ${upgradeRunTime}.`
3. For this failure: `fail("Total Runtime is not present in upgrade report.");` - it could also be present but not match upgrade.log. Maybe do 2 checks?
4. Why was `test.name.skip.portal.instance` removed from test DocumentLibraryLogged?
5. In the future, try to seperate SF changes from code logic changes. They are currently combined in https://github.com/vicnate5/liferay-portal/pull/2192/commits/b991bba5a521e849ab1a228c6a31aa247b14a8c6

One other tip: Use "ci test upgrade no-compile" when only testing poshi changes, it reduces CI runtime and CI load since we don't need to compile here. You can also add a temporary commit to reduce test runs to only what is new. See https://liferay.atlassian.net/wiki/spaces/ENGAPPSECURITY/pages/1881899867/How+to+run+only+selected+Poshi+test+cases+at+CI

**Comment by lucasperj** [comment]
ci:reopen

**Comment by lucasperj** [comment]
ci:test:upgrade

**Comment by vicnate5** [comment]
@lucasperj This branch is definitely improving, but I am going to ask for a few more changes.

1. Can you confirm that `echo("${fileContent}");` after the `fail()` is working? Try triggering the fail condition and see what happens.
2. Some properties like `property testcase.url` and `property skip.start.app.server` are set in every test, move them to the global properties.
3. Please fix the commits that add and remove extra rebase conflict output `<<<<<<< HEAD`. Use [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) to fix it. I normally use `e` to edit commits. Be good to practice splitting out the SF changes too.
4. Fix duplicated message "Upgrade runtime reported by upgrade.log".

Thanks for working on these!

**Comment by lucasperj** [comment]
ci:reopen

**Pull request closed without merging**